natss/config/200-addressable-resolver-clusterrole.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: natss-addressable-resolver
+  labels:
+    eventing.knative.dev/release: devel
```

Comment:
This label should be `contrib.eventing.knative.dev/release`

Reply:
@grantr thanks for the review. should it be changed to `release` in all occurrences?

Reply:
:+1: All release labels in this repo should look like this:

```yaml
labels:
  contrib.eventing.knative.dev/release: devel
```

The only difference is the addition of `contrib.` to the beginning of the label key.

Reply:
@grantr Gotcha! Thanks! ✌️